### PR TITLE
Add API to return manticore status information

### PR DESCRIPTION
**Pull request by RaedAlmomani**
Add API to return manticore status information.

### CODE REVIEW

**Review by crokita — CHANGES_REQUESTED** [review]
see feedback

File: api/v2/index.js [hunk 1]
```diff
@@ -71,6 +71,18 @@ module.exports = app => {
         ctx.response.body = status;
         ctx.response.status = 200;
     });
+    
+    //return manticore address information
+    router.get(`${API_PREFIX}/info`, async (ctx, next) => {
+        logger.debug(`GET ${API_PREFIX}/info`);
+        const wsAddress = await websocket.getPasscode();
```
[the remark this answers is not in the document]
Comment (RaedAlmomani) [reply api/v2/index.js:78]: Done!!

[the remark this answers is not in the document]
Comment (RaedAlmomani) [reply api/v2/index.js:78]: Done!!

File: api/v2/index.js [hunk 2]
```diff
@@ -71,6 +71,18 @@ module.exports = app => {
         ctx.response.body = status;
         ctx.response.status = 200;
     });
+    
+    //return manticore address information
+    router.get(`${API_PREFIX}/info`, async (ctx, next) => {
```
[the remark this answers is not in the document]
Comment (RaedAlmomani) [reply api/v2/index.js:76]: Done!!

File: api/v2/index.js [hunk 3]
```diff
@@ -71,6 +71,18 @@ module.exports = app => {
...
+        logger.debug(`GET ${API_PREFIX}/info`);
+        const wsAddress = await websocket.getPasscode();
+        ctx.response.status = 200;
+        ctx.response.body = {
+            path: `${API_PREFIX}/info/`,
+            protocol: 'ws',
+            passcode: wsAddress,
+        };
+    });
```
[the remark this answers is not in the document]
Comment (RaedAlmomani) [reply api/v2/index.js:86]: Done!

**Review by crokita — CHANGES_REQUESTED** [review]
See feedback

File: api/v2/app/interfaces/websocket/simple/index.js [hunk 4]
```diff
@@ -26,6 +26,13 @@ async function deletePasscode (id) {
     delete idHash[id];
 }
 
+//Verify the id's existence and return true if it does
+async function isIdExist (id) {
```
[the remark this answers is not in the document]
Comment (RaedAlmomani) [reply api/v2/app/interfaces/websocket/simple/index.js:30]: Done!

File: api/v2/index.js [hunk 5]
```diff
@@ -102,17 +129,8 @@ module.exports = app => {
...
-
-            if (config.modes.elb) { //ws addresses (ELB)
-                ctx.response.body.port = config.wsPort;
-            }
-            if (config.modes.elbEncryptWs) { //secure ws addresses (ELB)
-                ctx.response.body.protocol = 'wss';
-            }
-        }
+        if (config.modes.haproxy) 
+            ctx.response = updateCtxRespone(ctx.response);
```
Comment (crokita) [inline api/v2/index.js:133]:
```suggestion
        if (config.modes.haproxy) {
            ctx.response = updateCtxRespone(ctx.response);
        }
```

**Review by crokita — CHANGES_REQUESTED** [review]

**Review by crokita — APPROVED** [review]

**Pull request merged**